designer/server/src/main/scala/pl/touk/nussknacker/ui/process/newdeployment/synchronize/DeploymentsStatusesSynchronizationScheduler.scala

```diff
@@ -0,0 +1,48 @@
+package pl.touk.nussknacker.ui.process.newdeployment.synchronize
+
+import akka.actor.{ActorSystem, Cancellable}
+import com.typesafe.config.Config
+import net.ceedubs.ficus.Ficus._
+import net.ceedubs.ficus.readers.ArbitraryTypeReader._
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+// TODO: Properly handle HA setup: synchronizeAll() should be invoked only on one instance of designer in a time
```

Comment:
This is not a huge limitation. It would cause that DMs would be invoked twice instead of once. I want to handle it in separate PR because it is not so easy to do and it is not a priority ATM